dev/com.ibm.ws.jdbc_fat_oracle/fat/src/com/ibm/ws/jdbc/fat/oracle/FATSuite.java

```diff
@@ -30,83 +30,83 @@
-        String msg = frame.getUtf8String();
-        if (msg.endsWith("\n"))
-            msg = msg.substring(0, msg.length() - 1);
-        Log.info(FATSuite.class, "oracle", msg);
-    }
-
-    @BeforeClass
-    public static void beforeSuite() throws Exception {
-        //Allows local tests to switch between using a local docker client, to using a remote docker client.
-        ExternalTestServiceDockerClientStrategy.clearTestcontainersConfig();
-
-        oracle.start();
-    }
-
-    @AfterClass
-    public static void afterSuite() {
-        oracle.stop();
-    }
-
-    public static void initDatabaseTables() throws SQLException {
-        Properties connProps = new Properties();
-        // This property prevents "ORA-01882: timezone region not found" errors due to the Oracle DB not understanding
-        // some time zones(specifically those used by our RHEL 6 test systems).
-        connProps.put("oracle.jdbc.timezoneAsRegion", "false");
-
-        OracleDataSource ds = new OracleDataSource();
-        ds.setConnectionProperties(connProps);
-        ds.setUser(oracle.getUsername());
-        ds.setPassword(oracle.getPassword());
-        ds.setURL(oracle.getJdbcUrl());
-
-        try (Connection conn = ds.getConnection()) {
-            Statement stmt = conn.createStatement();
-
-            //Create MYTABLE for OracleTest.class and OracleTraceTest.class
-            try {
-                stmt.execute("DROP TABLE MYTABLE");
-            } catch (SQLException x) {
-                // probably didn't exist
-            }
-            stmt.execute("CREATE TABLE MYTABLE (ID NUMBER NOT NULL PRIMARY KEY, STRVAL NVARCHAR2(40))");
-
-            //Create CONCOUNT for OracleTest.class
-            try {
-                stmt.execute("DROP TABLE CONCOUNT");
-            } catch (SQLException x) {
-                // probably didn't exist
-            }
-            stmt.execute("CREATE TABLE CONCOUNT (NUMCONNECTIONS NUMBER NOT NULL)");
-            stmt.execute("INSERT INTO CONCOUNT VALUES(0)");
-
-            //Create COLORTABLE for OracleUCPTest.class
-            try {
-                stmt.execute("DROP TABLE COLORTABLE");
-            } catch (SQLException x) {
-                // probably didn't exist
-            }
-            stmt.execute("CREATE TABLE COLORTABLE (ID NUMBER NOT NULL PRIMARY KEY, COLOR NVARCHAR2(40))");
-            PreparedStatement ps = conn.prepareStatement("INSERT INTO COLORTABLE VALUES(?,?)");
-            ps.setInt(1, 1);
-            ps.setString(2, "maroon");
-            ps.executeUpdate();
-
-            //Close statements
-            ps.close();
-            stmt.close();
-        }
-    }
+	// TODO replace this container with the official oracle-xe container if/when it is available without a license
+	static OracleContainer oracle = new OracleContainer("kyleaure/oracle-18.4.0-xe-prebuilt:1.0")
+			.withExposedPorts(1521, 8080) // need to manually expose ports due to regression in 1.14.0
```

Comment:
The oracle container uses ports 1521, and 5500.
This regression will also affect fattest.databases.  You'll need to expose these ports here as well: 
https://github.com/OpenLiberty/open-liberty/blob/dfd64634f4f0715db178b558e88f52c9b74c8f05/dev/fattest.databases/src/componenttest/topology/database/container/DatabaseContainerFactory.java#L122-L124